redfish-exporter/.env

```diff
@@ -1,18 +1,20 @@
 UPDATED="2024-09-24"
 DESCRIPTION="Redfish Event Listener/Exporter"
-LISTENER_IP="0.0.0.0"
-LISTENER_PORT="8080"
+LISTENER_IP="10.11.18.55"
+LISTENER_PORT="9003"
 METRICS_PORT="2112"
 USE_SSL="false"
 CERTFILE="path/to/certfile"
 KEYFILE="path/to/keyfile"
-SLURM_USER="slurm user here"
-SLURM_TOKEN="token string here, from secret when for real"
-SLURM_CONTROL_NODE="slurm control node IP:Port"
+SLURM_CONTROL_NODE="10.235.34.47"
+SLURM_DRAIN_EXCLUDE_REASON_LIST="AMD|Pensando|RebootNeeded"
```

Comment:
The nodes with `reason/message` that matches one of these string will not be drained during critical event? 

Could you please add a comment?